src/demo/native/story.js

```diff
@@ -46,6 +46,68 @@ storiesOf('Demos', module)
 
     const colors = select('color', ['red', 'blue', 'white', 'black'], 'black');
 
+    window.onCallbackInput = ev => {
```

Comment:
Thats not a good idea. Do not set functions on `window` to not have collisions with other frameworks. Its a good practice to not touch the global window object.

Reply:
Hmm I used this a template -> https://github.com/axa-ch/patterns-library/pull/1387/files#diff-ad77956675a7781a5892ae6169b513a6R75
How is this going to affect other frameworks. It is only a story. So it doesn't pollutes the components core...(as far I know)

Reply:
agree generally with @danyball. Here however we have a scoped window object: is inside a story that will never go onto our clients Browser and is inside a `iframe` in our storybook.

native `onclick` or `onchange` always require a window function. If you really want to avoid it you have to add a listener. I personally think is a good idea to test here in the story the native `onclick` handler